packages/monitor-v2/src/monitor-polymarket/MonitorProposalsOrderBook.ts

```diff
@@ -118,7 +137,10 @@ export async function monitorTransactionsProposedOrderBook(
         },
         params
       );
-      notifiedProposals.push(market);
+      if (!notified) {
```

Comment:
If a market proposal correspond to a high volume market and also the market is signalling a wrong price we allow the bot to notify twice (with different messages) but we only push the market once to `notifiedProposals`